src/opera/pge/dswx_ni/dswx_ni_pge.py

```diff
@@ -345,7 +345,7 @@ class DSWxNIExecutor(DSWxNIPreProcessorMixin, DSWxNIPostProcessorMixin, PgeExecu
     LEVEL = "L3"
     """Processing Level for DSWx-NI Products"""
 
-    SAS_VERSION = "0.1"
+    SAS_VERSION = "0.2"
```

Comment:
For traceability, I like to include a link to the github release for the associated SAS here, so please add this as an inline comment:

`# Beta release https://github.com/opera-adt/DSWX-SAR/releases/tag/DSWx-NI-v0.2`